Public/Src/Engine/Scheduler/Tracing/Log.cs

```diff
@@ -263,6 +263,15 @@ internal static string PipWriteFileFailedMessage(string pipDescription, string p
             Message = "[{pipDescription}] Cache miss (fingerprint '{fingerprint}'): Process will be executed.")]
         internal abstract void ScheduleProcessPipCacheMiss(LoggingContext loggingContext, string pipDescription, string fingerprint);
 
+        [GeneratedEvent(
+            (ushort)EventId.ProcessPipProcessWeight,
+            EventGenerators = EventGenerators.LocalOnly,
+            EventLevel = Level.Verbose,
+            Keywords = (int)Events.Keywords.UserMessage,
+            EventTask = (ushort)Events.Tasks.Storage,
+            Message = "[{pipDescription}] Executing process with process weight: {weight}.")]
+        internal abstract void ProcessPipProcessWeight(LoggingContext loggingContext, string pipDescription, int weight);

```

Comment:
>ProcessPipProcessWeight [](start = 31, length = 23)

Do we not have an executing process message already that this can be added to?

Reply:
Actually, no not really. Some messages related to preparing to execute a pip and some messages related to post-execution, but nothing for just executing.

---
In reply to: [271528643](https://github.com/Microsoft/BuildXL/pull/67#discussion_r271528643) [](ancestors = 271528643)